graphql_compiler/cost_estimation/filter_selectivity_utils.py

```diff
@@ -450,6 +450,16 @@ def get_selectivity_of_filters_at_vertex(schema_info, filter_infos, parameters,
     return combined_selectivity
 
 
+def adjust_counts_with_selectivity(counts, selectivity):
+    """Apply the selectivity to the unfiltered vertex count and return the result."""
```

Comment:
Can you give some more context here, through the name of the function, the documentation, or the naming of the input variables (and ideally all of them)? This is all quite opaque.

Also, missing type hints.